applications/ChimeraApplication/python_scripts/navier_stokes_solver_vmsmonolithic_chimera.py

```diff
@@ -57,6 +57,9 @@ def Initialize(self):
             self.model,
             self.chimera_internal_parts)
 
+    def GetComputingModelPart(self):
+        return self.main_model_part
```

Comment:
Just for me to know. Why do you need the `main_model_part`? It is because it contains both the background mesh and the patches?

Reply:
not exactly. I need to use the `main_model_part` because the constraints are added on this modelpart and `computing_model_part` does not know anything about them.

Reply:
mmmm now I see. Do you think we should add them to the computing model part too?

Reply:
In the current impelementations, not necessary. But as I remember that computing modelpart is set to be removed in future no ?

Reply:
In the structure. It is required to have it in the fluid.

Reply:
Is it really required in FluidApp ? If not we can also try to remove it !

Reply:
I discussed this many times with @jcotela, he said it is not possible to remove it
Not sure if this has changed